src/FM.LiveSwitch.Mux.Standard/Muxer.cs

```diff
@@ -102,6 +102,10 @@ public async Task<bool> Run()
                         Options.InputFileNames = string.Join(",", Options.InputFileNames).Split(',').Where(fileName => fileName.Length > 0);
                     }
 
+                    var preprocessor = new JsonPreprocessor(_Logger, Options.InputPath);
+                    preprocessor.MinimumOrphanDuration = Options.MinimumOrphanDuration;
+                    await preprocessor.processDirectory();
```

Comment:
Method names could be UpperCamelCase, so `ProcessDirectory` not `processDirectory`.